tgui/packages/tgui/interfaces/OverwatchConsole.js

```diff
@@ -0,0 +1,860 @@
+import { useBackend, useLocalState } from '../backend';
+import { round } from 'common/math';
+import {
+  Button,
+  Section,
+  NoticeBox,
+  Stack,
+  Flex,
+  Icon,
+  NumberInput,
+  LabeledList,
+  ProgressBar,
+  Divider,
+  Table,
+  Box,
+  Dropdown,
+  ColorBox,
+} from '../components';
+import { Window } from '../layouts';
+
+import { createLogger } from '../logging';
+const logger = createLogger('Overwatch');
+
+export const OverwatchConsole = (_props, context) => {
+  const { act, data } = useBackend(context);
+  const { operator } = data;
+  const body = operator ? <OverwatchMain /> : <OverwatchEmpty />;
+  const windowWidth = operator ? 1000 : 600;
+  const windowHeight = operator ? 700 : 300;
+
+  return (
+    <Window
+      width={windowWidth}
+      height={windowHeight}
+      scrollable
+      theme="weyland">
+      <Window.Content>{body}</Window.Content>
+    </Window>
+  );
+};
+
+export const OverwatchMain = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { squad_data } = data;
+  const [currentControlCategory] = useLocalState(
+    context,
+    'current_control_category',
+    1
+  );
+
+  return (
+    <>
+      <OverwatchId />
+      {squad_data ? (
```

Comment:
Conditional rendering again. You have a fair bit of component code which is in this conditional segment. By the time you get to the false condition statement you need to scroll up to find out what the condition is.